libc/test/src/sys/epoll/linux/epoll_create1_test.cpp

```diff
@@ -0,0 +1,32 @@
+//===-- Unittests for epoll_create1 ---------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+#include "src/errno/libc_errno.h"
+#include "src/sys/epoll/epoll_create1.h"
+#include "src/unistd/close.h"
+#include "test/UnitTest/ErrnoSetterMatcher.h"
+#include "test/UnitTest/Test.h"
+
+#include <sys/epoll.h>
```

Comment:
Why do you pull in this system header, for this test and the ones below?

Reply:
So does your newly added libc/include/llvm-libc-macros/linux/sys-epoll-macros.h, why not use that for this test, rather than the system header?